script/apps/Aries/Creator/Game/Entity/EntityCode.lua

```diff
@@ -440,8 +440,11 @@ end
 
 -- called when the user clicks on the block
 -- @return: return true if it is an action block and processed . 
-function Entity:OnClick(x, y, z, mouse_button, entity, side)
-	if(GameLogic.isRemote) then
+function Entity:OnClick(x, y, z, mouse_button, entity, side)	
+	local canBeOpened = GameLogic.GetFilters():apply_filters("CustomCodeBlockClicked", false, mouse_button, entity);
```

Comment:
better call open editor in the filter and totally skip code. 

if(GameLogic.GetFilters():apply_filters("CustomCodeBlockClicked", false, mouse_button, entity)) then
   return true
end